webapp/store/views.py

```diff
@@ -269,9 +273,8 @@ def snap_details(snap_name):
         if not details.get("channel-map"):
             flask.abort(404, "No snap named {}".format(snap_name))
 
-        formatted_paragraphs = logic.split_description_into_paragraphs(
-            details["snap"]["description"]
-        )
+        clean_description = bleach.clean(details["snap"]["description"])
+        formatted_description = parse_markdown_description(clean_description)
```

Comment:
Would it make sense to use `bleach.clean` on result of `parse_markdown_description`? This way we would be sanitizing HTML rather then Markdown.

Reply:
This may actually help getting rid of unwanted HTML (like `<img>`), because even if Markdown produces image it will be removed by bleach.

Reply:
@bartaz the problem is that the `clean` method escapes some tags that I don't want (`<p>` for ex) that are generated by the mardown parser...

Reply:
Ah, right. By default it only allows inline styling elements.

But, as we are so careful about what we allow and what not - we could easily create a custom config for bleach as well? To list what we allow from Markdown and what not?

Reply:
I am not sure it's the right solution, I don't really want to add custom rules to every library that we use. It will be quite confusing in the future to maintain some custom stuff that we add...